example/ios_example/App.swift

```diff
@@ -1,18 +1,135 @@
 import Authgear
 import SwiftUI
 
-class App: ObservableObject {
-    @Published var container: Authgear?
-    let appState: AppState
-    let mainViewModel: MainViewModel
-
-    init() {
-        appState = AppState()
-        mainViewModel = MainViewModel(appState: appState)
-    }
+struct UserInfo {
+    var userID: String
+    var isAnonymous: Bool
+    var isVerified: Bool
+}
 
+class App: ObservableObject {
     static let redirectURI = "com.authgear.example://host/path"
     static let weChatUniversalLink = "https://authgear-demo.pandawork.com/wechat/"
     static let weChatRedirectURI = "https://authgear-demo.pandawork.com/authgear/open_wechat_app"
     static let weChatAppID = "wxa2f631873c63add1"
+
+    @Published var container: Authgear?
+    @Published var user: UserInfo?
+    @Published var page: String = ""
+    @Published var authgearActionErrorMessage: String?
+    @Published var successAlertMessage: String?
+
+    func configure(clientId: String, endpoint: String, isThirdParty: Bool, page: String) {
+        guard clientId != "", endpoint != "" else {
+            authgearActionErrorMessage = "Please input client ID and endpoint"
+            return
+        }
+        guard let appDelegate = UIApplication.shared.delegate as? AppDelegate else {
+            authgearActionErrorMessage = "Failed to configure Authgear"
+            return
+        }
+        UserDefaults.standard.set(clientId, forKey: "authgear.demo.clientID")
+        UserDefaults.standard.set(endpoint, forKey: "authgear.demo.endpoint")
+        UserDefaults.standard.set(page, forKey: "authgear.demo.page")
+        UserDefaults.standard.set(!isThirdParty, forKey: "authgear.demo.isFirstParty")
+        appDelegate.configureAuthgear(clientId: clientId, endpoint: endpoint, isThirdParty: isThirdParty)
+        successAlertMessage = "Configured Authgear successfully"
+        self.page = page
+    }
+
+    private func handleAuthorizeResult(_ result: Result<AuthorizeResult, Error>, errorMessage: String) -> Bool {
+        switch result {
+        case let .success(authResult):
+            let userInfo = authResult.userInfo
+            user = UserInfo(
+                userID: userInfo.sub,
+                isAnonymous: userInfo.isAnonymous,
+                isVerified: userInfo.isVerified
+            )
+            return true
+        case let .failure(error):
+            print(error)
+            authgearActionErrorMessage = errorMessage
+            return false
+        }
+    }
+
+    func login(container: Authgear?) {
+        container?.authorize(
+            redirectURI: App.redirectURI,
+            prompt: "login",
```

Comment:
I think we should remove the prompt parameter in the example, so that examples in all platform have the same login behavior. I removed it in commit 3f2206b0